csm/cli/schema/system.json

```diff
@@ -129,6 +129,7 @@
         "table": {
           "headers": {
             "name": "Node ID",
+            "hostname": "Hostname",
```

Comment:
Will User Parse srv node for start/stop or hostname??

Reply:
User will Parse  both srv node as well as hostname. As user can also trigger start/stop with hostname. Hostname will be mapped to its corresponding srvnode id and that srvnode id will be used for further operations.

Reply:
Please Update the commands as well since the provide specific for node name and add in Help that either node_name or hostname both will work

Reply:
Comment resolved .Please review